html/components/spinners.js

```diff
@@ -84,6 +90,37 @@ const spinners = () => {
       }
     });
   });
+
+  /**
+   * This selector is exactly the same as data-sprk-spinner="click" except
+   * the disabled attribute will not be added to the button when it is
+   * spinning.
+   */
+  getElements('[data-sprk-spinner="is-not-disabled"]', (spinnerContainer) => {
```

Comment:
is this implying that it either has `click` or `is-not-disabled`? 

What are you thoughts on if we grabbed elements with [data-sprk-spinner] and then do an in statement based on if its value is `click` or `is-not-disabled`so we're repeating less code?

Reply:
Perhaps something like this: https://stackoverflow.com/questions/7084557/select-all-elements-with-a-data-xxx-attribute-without-using-jquery

and then using `getAttribute` to switch between the two